src/main/java/frc/trigon/robot/subsystems/swerve/SwerveConstants.java

```diff
@@ -39,30 +40,18 @@ public class SwerveConstants {
             REAR_LEFT_WHEEL_DIAMETER_METERS = RobotHardwareStats.isSimulation() ? 0.1016 : 0.049274 * 2,
             REAR_RIGHT_WHEEL_DIAMETER_METERS = RobotHardwareStats.isSimulation() ? 0.1016 : 0.049274 * 2;
```

Comment:
1. get these values from the config 
2. we'll drop the individual diameter stuff

Reply:
why we dropping it?

Reply:
for the first point, and because it really isn't that helpful